Backend/ServiceLayer/Services/EmailService.cs

```diff
@@ -24,7 +24,7 @@ public class EmailService: IEmailService
         public MimeMessage createEmailPlainBody(string receiverName, string receiverEmail, string emailSubject, string emailBody)
         {
             var message = new MimeMessage();
-            message.From.Add(new MailboxAddress("Support", "support@kfcsso.com"));
+            message.From.Add(new MailboxAddress("Support", "support@greetngroup.com"));
```

Comment:
Reason for this?

Reply:
Can you make it like 'noreply-support@kfcsso.com'?

Reply:
SES for the website will need to be setup for it to work